code/processes/wdb.q

```diff
@@ -183,7 +184,7 @@ endofday:{[pt]
 	.lg.o[`eod;"end of day message received - ",spt:string pt];	
 	/- create a dictionary of tables and merge limits
 	mergelimits:(tablelist[],())!({[x] mergenumrows^mergemaxrows[x]}tablelist[]),();	
-  tablist:tablelist[]!{0#value x} each tablelist[];
+    tablist:tablelist[]!{0#value x} each tablelist[];
```

Comment:
why the whitespace change? looks like if anything it should have gone the other direction to line up with above (might be a tabs/spaces issue)

Reply:
I was looking at this right now. It seems that someone used tab in the wdb.q. The alignment is a bit off in that file. I'll see if I can fix it